MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -71,6 +71,21 @@ public class EncodingHelper
             "m4v",
         };
 
+        // Set max transcoding channels for encoders that can't handle more than a set amount of channels
+        // AAC, FLAC, ALAC, libopus, libvorbis encoders all support at least 8 channels
+        private static readonly Dictionary<string, int> _audioTranscodeChannelLookup = new(StringComparer.OrdinalIgnoreCase)
+        {
+            { "wmav2", 2 },
+            { "libmp3lame", 2 },
+            { "libfdk_aac", 6 },
+            { "aac_at", 6 },
+            { "ac3", 6 },
+            { "eac3", 6 },
+            { "dts", 6 },
```

Comment:
```suggestion
            { "dca", 6 },
```

This is wrong in other places too (see 87ba19aaa67292838038bd0df58a5e26f41c8821)